sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusManagementAsyncClient.java

```diff
@@ -593,8 +595,9 @@
             return new SimpleResponse<>(response.getRequest(), response.getStatusCode(), response.getHeaders(), null);
         }
 
-        final QueueDescription result = entry.getContent().getQueueDescription()
-            .setName(entry.getTitle().getTitle());
+        final QueueDescription result = entry.getContent().getQueueDescription();
```

Comment:
Is this going to be always not null `entry.getContent()` ? Could this throw NPE ?